Example Apps/Example Swift/PerformInteractionManager.swift

```diff
@@ -21,7 +21,8 @@ class PerformInteractionManager: NSObject {
     ///
     /// - Parameter manager: The SDL Manager
     func show(from triggerSource: SDLTriggerSource) {
-        manager.screenManager.presentSearchableChoiceSet(choiceSet, mode: interactionMode(for: triggerSource), with: self)
+        //manager.screenManager.presentSearchableChoiceSet(choiceSet, mode: interactionMode(for: triggerSource), with: self)
+        manager.screenManager.present(choiceSet, mode: .voiceRecognitionOnly)
```

Comment:
Uncomment the commented out code and remove current `present`

Reply:
done